contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -51,9 +52,10 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   uint256 internal s_fallbackLinkPrice; // not in config object for gas savings
   uint96 internal s_ownerLinkBalance;
   uint256 internal s_expectedLinkBalance;
-  address internal s_transcoder;
-  address internal s_registrar;
+  address internal s_transcoder; // use interface as type
+  address internal s_registrar; // cannot use interface as type bc only use case is to check if msg.sender == s_registrar
 
+  // all the interfaces should be named like: ILinkToken, IAggregatorV3
```

Comment:
Let's not change this now as it would be a much bigger change in multiple unrelated files

Reply:
OK. this can be a TODO

Reply:
Yeah i think there's a bigger effort already going on to rename all interfaces with I for the whole repo

Reply:
I second @infiloop2 's point here. Please use the existing naming format for now.